modules/gdscript/gdscript_warning.cpp

```diff
@@ -123,7 +123,7 @@ String GDScriptWarning::get_message() const {
 		} break;
 		case UNSAFE_CALL_ARGUMENT: {
 			CHECK_SYMBOLS(4);
-			return "The argument '" + symbols[0] + "' of the function '" + symbols[1] + "' requires a the subtype '" + symbols[2] + "' but the supertype '" + symbols[3] + "' was provided";
+			return "The argument " + symbols[0] + " of the function '" + symbols[1] + "' requires a the subtype '" + symbols[2] + "' but the supertype '" + symbols[3] + "' was provided.";
```

Comment:
Is it possible to change this to `vformat` which is supposed to be faster?

Reply:
Yes, I will clean up all the warnings in a separate PR (`vformat`, double quotes, trailing dot, order).

Reply:
* Done: #76412.